script/rollout.py

```diff
@@ -0,0 +1,69 @@
+import pickle
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+import torch
+import torch.optim as optim
+from torch.utils.data import DataLoader
+import sys
+from model import *
+from prep_data import *
+
+""" After training the model, we can try to use the model to do
+jumpy predictions.
+"""
```

Comment:
Does this string help describe what this file will do? Consider moving it to a file/top-level docstring (before the imports) to describe the purpose along with any notes.